manifests/params.pp

```diff
@@ -105,26 +105,29 @@
       $service_restart      = 'pkill -HUP telegraf'
     }
   }
-  $package_name           = 'telegraf'
-  $ensure                 = 'present'
-  $install_options        = []
-  $hostname               = $trusted['hostname']
-  $omit_hostname          = false
-  $interval               = '10s'
-  $round_interval         = true
-  $metric_batch_size      = 1000
-  $metric_buffer_limit    = 10000
-  $collection_jitter      = '0s'
-  $flush_interval         = '10s'
-  $flush_jitter           = '0s'
-  $precision              = ''
-  $debug                  = false
-  $quiet                  = false
-  $global_tags            = {}
-  $manage_service         = true
-  $purge_config_fragments = false
-  $repo_type              = 'stable'
-  $windows_package_url    = 'https://chocolatey.org/api/v2/'
+  $package_name                  = 'telegraf'
+  $ensure                        = 'present'
+  $install_options               = []
+  $hostname                      = $trusted['hostname']
+  $omit_hostname                 = false
+  $interval                      = '10s'
+  $round_interval                = true
+  $logfile_rotation_interval     = '0h'
+  $logfile_rotation_max_size     = '0MB'
+  $logfile_rotation_max_archives = 5
```

Comment:
Instead of adding the data here, please directly assign it in the init.pp. That enables puppet-strings to pick it up.

Reply:
Not sure that is the best way, I did it this way to be 100% consistent with the existing code, e.g. logfile.
In fact all values in init.pp reference telegraf::params

Reply:
yes it's the best way :) That's the standard we defined at https://voxpupuli.org/docs/reviewing_pr/  and we need to migrate the modules. We don't want more technical debt.